BSP/Simulator_V2/Src/BSP_CAN.c

```diff
@@ -1,25 +1,35 @@
 /* Copyright (c) 2022 UT Longhorn Racing Solar */
 
 #include "BSP_CAN.h"
-#include "simulator_conf.h"
+#include "Simulator.h"
 #include <stdint.h>
 #include <stdlib.h>
 #include <sys/file.h>
 #include <unistd.h>
 
+#define to64l(arr) (((uint64_t)(((uint8_t *)(arr))[0]) <<  0)+\
+                    ((uint64_t)(((uint8_t *)(arr))[1]) <<  8)+\
+                    ((uint64_t)(((uint8_t *)(arr))[2]) << 16)+\
+                    ((uint64_t)(((uint8_t *)(arr))[3]) << 24)+\
+                    ((uint64_t)(((uint8_t *)(arr))[4]) << 32)+\
+                    ((uint64_t)(((uint8_t *)(arr))[5]) << 40)+\
+                    ((uint64_t)(((uint8_t *)(arr))[6]) << 48)+\
+                    ((uint64_t)(((uint8_t *)(arr))[7]) << 56))
```

Comment:
Tbh, I don't think you need to send data like this. You don't need to mimic CAN to the basics since this just means when we log the data, we will have to do some extra parsing to reverse it. Assume that the CAN protocol works correctly and just send without reversing.

Reply:
Ok, will change this